README.md

```diff
@@ -1,10 +1,11 @@
+# Notice: The next version of this project can be found at [videojs-http-streaming](https://github.com/videojs/http-streaming)
```

Comment:
It may be worth pointing to the VJS 7 blog post or have some short description of the reason why it's the next version.